### PR TITLE
Added nullable fields for Lab Duplicates and Lab Blanks. 

### PR DESCRIPTION
**Pull request by TylerGoulskiERG**
Ticket: https://github.com/Eastern-Research-Group/AquaQAPP/issues/446

Lab Duplicates and Lab Blanks can now be left blank in side nav. Edit button now open corresponding data in sidenav

### CODE REVIEW

File: client/src/components/app/SampleDesign.vue [hunk 1]
```diff
@@ -132,6 +132,14 @@ export default {
           key: 'parameterLabel',
           label: 'Parameter/Method',
         },
+        {
```
Comment (dpgalloway) [inline client/src/components/app/SampleDesign.vue]: I don't think these two new columns were intended to be committed. For now let's just display the Location ID and Parameter columns

File: client/src/components/app/SampleDesign.vue [hunk 2]
```diff
@@ -202,16 +210,18 @@ export default {
       const locationQuestions = this.$store.state.structure.questions.filter(
         (q) => q.section.sectionLabel === 'Monitoring Locations'
       );
-      const relevantQuestions = locationQuestions.concat(this.questions);
+      // const relevantQuestions = locationQuestions.concat(this.questions);
```
Comment (dpgalloway) [inline client/src/components/app/SampleDesign.vue]: Please go ahead and just delete this line

**Review by dpgalloway — CHANGES_REQUESTED** [review]

File: client/src/components/app/SampleDesign.vue (hunk 2 above)
Comment (TylerGoulskiERG) [reply client/src/components/app/SampleDesign.vue]: Line removed in commit f623f40

File: client/src/components/app/SampleDesign.vue (hunk 1 above)
Comment (TylerGoulskiERG) [reply client/src/components/app/SampleDesign.vue]: Removed columns in commit f623f40

**Review by dpgalloway — APPROVED** [review]

**Pull request merged**